### DISCUSSION TITLE
implementation of peripheral venous blood in Mobi for PD modeling

### DISCUSSION THREAD

**Discussion by CK-CHO** [Open-Systems-Pharmacology/Forum#886] [Questions & Problems] [answered]
Hello, i have a questions about implementation of peripheral venous plasma concentration. 

I didn't find the peripheral venous plasma concentration in possible referenced objects in new parameters of spatial structures. Provided peripheral venous blood compartment were not clicked in the software.

![PD 질문 1](https://user-images.githubusercontent.com/49744058/114803558-9e590e80-9dda-11eb-8221-b983146fe8b9.png)

How can i incorporating the plasma concentration-time profiles in the PD model?

When i used the table (multiple time discrete and piecewise constant numeric values) in specific time point nor the formula (an explicit formula) values, i confirmed that PD model is plausible.  So, i want to incorporate the simulated profiles in the PD model, not the specific table for improvement of our model. 

![PD 질문 3](https://user-images.githubusercontent.com/49744058/114804067-9c437f80-9ddb-11eb-859c-40f84d6a769b.png)

Please give me the opinion and help. 

Thank you.

**Comment by PavelBal** [comment, marked as answer]
Hi,

the "PeripheralVenousBlood" is an [observer](https://docs.open-systems-pharmacology.org/working-with-mobi/mobi-documentation/model-building-components#observers). It is mixture of the concentrations in Muscle and Fat plasma. Observers cannot be used in equations.
Instead, you can use the "VenousBlood|Plasma" concentrations, which should be very close to the PeripheralVenousBlood unless your compound is extensively metabolized in muscle and/or fat.
